### DISCUSSION TITLE
How to manage links when localizing content.

### DISCUSSION THREAD

**Discussion by michaelcrook** [mkdocs/mkdocs#2533] [Q&A] [answered]
The team that I work on is looking at how we could localize the markdown files in our mkdocs-material deployment. One of the concerns is that the links coming for the software into the documentation will be different based on the language used. Is there a way to create a singular link name/id for a header that will work across all localized languages?

**Comment by lampensau** [comment, marked as answer]
You can use the [attr_list](https://python-markdown.github.io/extensions/attr_list/) extension to define custom slugs. (#2085)

After [enabling](https://www.mkdocs.org/user-guide/configuration/#markdown_extensions), you can just add `{ #custom-ID }` to your headings to overwrite the generated slugs (#links).

**Comment by ultrabug** [comment]
@michaelcrook did you have a look at [the mkdocs-static-i18n plugin?](https://github.com/ultrabug/mkdocs-static-i18n).

It handles link and reference localization automatically for you when you use it. I [explained and demonstrated everything here](https://ultrabug.fr/Tech%20Blog/2021/2021-07-28-create-beautiful-and-localized-documentations-and-websites-using-mkdocs-github/) if you want to give it a try.
